app/src/main/java/edu/uc/hickmadc/customnotifications/ui/main/MainFragment.kt

```diff
@@ -5,57 +5,27 @@ import androidx.fragment.app.Fragment
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
-import android.widget.ImageButton
-import android.widget.Toast
-import androidx.lifecycle.ViewModelProvider
 import edu.uc.hickmadc.customnotifications.R
-import android.widget.ListView
-
-
 
 class MainFragment : Fragment() {
 
-
- //temp data
-   private val language = arrayOf<String>("Brush Teeth","Eat Breakfast","Do schoolwork")
-   private val description = arrayOf<String>(
-        "Brush teeth for 2 minutes",
-        "Eat so you don't feel like shit",
-        "")
-   private val days= arrayOf<String>("MWF","TFS","WTF")
-   private val times= arrayOf<String>("1:30","13:15","8:30")
-
     companion object {
         fun newInstance() = MainFragment()
     }
 
     private lateinit var viewModel: MainViewModel
 
-
     override fun onCreateView(
-        inflater: LayoutInflater,
-        container: ViewGroup?,
+        inflater: LayoutInflater, container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View {
         return inflater.inflate(R.layout.fragment_main, container, false)
-
     }
 
-    override fun onActivityCreated(savedInstanceState: Bundle?) {
-        super.onActivityCreated(savedInstanceState)
-        viewModel = ViewModelProvider(this).get(MainViewModel::class.java)
-
-          val addButton = view?.findViewById<ImageButton>(R.id.btnAdd)
-                addButton?.setOnClickListener{
-                    var dialog= dialogfragment()
-
-                    dialog.show(requireActivity().supportFragmentManager!!, "customdialog")
-            }
-
-        val lView= requireView().findViewById<ListView>(R.id.listView)
-        val myListAdapter = MyListAdapter(requireActivity(),language,description,days, times)
-        lView.adapter = myListAdapter
-
-    }
+//    override fun onActivityCreated(savedInstanceState: Bundle?) {
+//        super.onActivityCreated(savedInstanceState)
+//        viewModel = ViewModelProvider(this).get(MainViewModel::class.java)
+//        // TODO: Use the ViewModel
+//    }
```

Comment:
Committing commented out code is considered bad practice in version control, because version control offers better alternatives:
1) Use a feature branch for experimental features.
2) Delete what is not needed.  You can always compare with history to see what used to be there.
Commented out code is considered clutter in a version control system.